datasets/us/ia/med_exclusions/us_ia_med_exclusions.yaml

```diff
@@ -37,3 +37,9 @@ assertions:
     schema_entities:
       Organization: 400
       Person: 1000
+
+lookups:
+  sanction_end_date:  
+    options:  
+      - match: "2 Years"
+        value: 2026-08-06
```

Comment:
perhaps this needs to be `max(entity.get(start_date)) + timedelta(years=2)` and not 2 years after the start of the row that triggered this.